Framework/jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/SeleniumConfig.java

```diff
@@ -421,6 +423,22 @@ private static DesiredCapabilities getRemoteCapabilities() {
     return capabilities;
   }
 
+  /**
+   * Get the remote capabilities as a HashMap
+   * @return HashMap of remote capabilities
+   */
+  public static HashMap<String, String> getRemoteCapabilitiesAsStrings() {
```

Comment:
> The return type of this method should be an interface such as "Map" rather than the implementation "HashMap". - From SonarCloud

See if that is possible to do.